### DISCUSSION TITLE
[Script request] pocket-id

### DISCUSSION THREAD

**Discussion by adaszko** [community-scripts/ProxmoxVE#729] [Request script]
### Application Name

pocket-id

### Website

https://github.com/stonith404/pocket-id

### Description

A simple and easy-to-use OIDC provider that allows users to authenticate with their passkeys to your services.

### Due Diligence

- [X] I have searched existing [scripts](https://community-scripts.github.io/Proxmox/scripts) and found no duplicates.
- [X] I have searched existing [discussions](https://github.com/community-scripts/ProxmoxVE/discussions?discussions_q=) and found no duplicate requests.
- [X] This is not a game-related request.

**Comment by Node815** [comment]
I will say that I use this as a Docker image and it totally replaced my Authentik server.  I think that using Webuthn (passkey) is the future of the no password world.  I use it every service that I host which supports OIDC integration.   

The developer is super active and responsive, which means, frequent releases come in different bursts.  The releases are roughly every 2 weeks or more depending on the activity.  Might be slower with the holidays, but I don't know if this would be a good candidate if it's being updated so frequently.   So this may be a consideration for whomever evaluates this request.

**Comment by Aloe-recite** [comment]
Seems great! Looks nice, and seems fast and simple! Easy to grasp apparently, and probably more suited to a 1 or 2 persons setup like main unlike Authentik and the rest which seem far too powerful and complex. Thanks!

**Comment by adaszko** [comment]
Looks like it landed in https://github.com/community-scripts/ProxmoxVE/pull/1779
